src/bp-core/bp-core-update.php

```diff
@@ -3401,3 +3405,19 @@ function bb_update_to_2_4_60() {
 
 	bb_create_background_member_friends_count();
 }
+
+/**
+ * Clear web and api cache on the update.
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @return void
+ */
+function bb_update_to_2_4_70() {
+	wp_cache_flush();
+	// Purge all the cache for API.
+	if ( class_exists( 'BuddyBoss\Performance\Cache' ) ) {
+		// Clear members API cache.
+		BuddyBoss\Performance\Cache::instance()->purge_by_component( 'bp-members' );
+	}
+}
```

Comment:
remove this, we don't want to clear the API cache on update.
We just need to clear the cache when user changes the settings for "Default Profile Avatar" or "Default Group Avatar"
https://prnt.sc/jYPYMqk0uAkX